instrumentation/events/lttng-module/net.h

```diff
@@ -8,7 +8,106 @@
 #include <linux/skbuff.h>
```

Comment:
comment about the changelog: please add a babeltrace output of a few sample events generated by this tracepoint.